src/style.css

```diff
@@ -2,2 +1,78 @@
+html,
+body,
+.header,
+.img-link,
+.nav-link,
+.nav-list {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', 'Arial', sans-serif;
```

Comment:
Good job on using fallback fonts. This is important in case the main one doesn't work. However, remember to always add a generic font family at the end of the list. In this case, 'Arial' is a specific font, not a generic one. A generic font could be 'serif' or 'sans-serif'.